app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::ApplicationController < ActionController::Base
+  include CurrentUser
+  
+  def login_user
+    warden.authenticate(:basic) || unauthorized!
```

Comment:
how about `ALLOWED_AUTH_METHODS = [:basic]` so it's more explicit and less copy-paste ?